optuna/trial.py

```diff
@@ -184,7 +187,9 @@ def suggest_discrete_uniform(self, name, low, high, q):
         """Suggest a value for the discrete parameter.
 
         The value is sampled from the range ``[low, high]``, and the step of discretization is
-        ``q``.
+        ``q``. More specifically, the returned value are ``low, low + q, low + 2 * q, ... ,
```

Comment:
"are" in this sentence seems a grammatical mistake.
I think, for example, it can be corrected as follows:
- "the possible return values are ``low, ...``", or
- "one of the values in the sequence (set?) ``low, ...`` is returned", or
- "the return value is sampled from the sequence ``low, ...``", or 
- etc

Reply:
You're right. I've updated the sentence as follows:
> More specifically, this method returns one of the values in the sequence ``low, low + q, low + 2 * q, ..., low + k * q <= high``,

Reply:
👍